vizro-core/docs/pages/tutorials/explore-components.md

```diff
@@ -459,47 +459,17 @@ You can apply selectors to configure [`Filters`][vizro.models.Filter] and [`Para
 - [`Slider`][vizro.models.Slider]
 
 ## 4. The final touches
+Each page is added to the dashboard using the following line of code: `vm.Dashboard(pages=[first_page, second_page])`. 
+This ensures that all the pages are accessible.
 
-This section puts everything together by adding a homepage to the example for navigation between the two separate pages.
+By default, a navigation panel is added on the left side to switch between the two pages. 
 
-For easy navigation within your dashboard, we'll create a page that serves as the entry point for the user. On this homepage are two [`Cards`][vizro.models.Card] which serve as tiles that can be customized with a title, some text, and an image. These cards link to the subpages within your dashboard using their `href` attributes as `href="/first-page"` and `href="/second-page"`. This establishes the navigation links from the homepage to each of the subpages.
```

Comment:
@stichbury - just FYI why I deleted all this: I am going to rewrite the entire tutorial at some point, as the content is quite outdated. This was written before the majority of features were in. I've removed the entire creation of the homepage, because, given the new functionality of our navigation, it doesn't really make sense to create a homepage here. Hence, I would just mention that a default navigation panel is added. In the future I'll enhance this section by showing how to customize the navigation, but for now, let's just leave it out.

Reply:
Thanks for the explanation. This all looks good to me.